Makefile

```diff
@@ -46,13 +28,25 @@ build-dev:
 
 mkfile_path := $(abspath $(lastword $(MAKEFILE_LIST)))
 
-run-dev:
-	docker run --interactive --tty --volume=$(dir $(mkfile_path))openslides_backend:/srv/code/openslides_backend \
+dev_args=--interactive --tty \
+	--volume=$(dir $(mkfile_path))openslides_backend:/srv/code/openslides_backend \
+	--volume=$(dir $(mkfile_path))tests:/srv/code/tests \
 	--network=host --rm openslides-backend-dev
 
-run-dev-interactive:
-	docker run --interactive --tty --volume=$(dir $(mkfile_path))openslides_backend:/srv/code/openslides_backend \
-	--network=host --rm openslides-backend-dev sh
+run-dev:
+	docker run $(dev_args)
+
+run-dev-interactive run-bash:
+	docker run $(dev_args) sh
+
+run-cleanup:
+	docker run $(dev_args) sh cleanup.sh
+
+run-tests:
```

Comment:
A dependency like `build-dev` is missing. same for `run-cleanup`.

Reply:
As far as I can tell the dependency is missing everywhere.

Reply:
Yes, these weren't there beforehand. I assumed @normanjaeckel didn't want that dependency to be executed on every run but to have manual control over it. But if he agrees, I can sure add it

Reply:
At least for `run-tests` (a required target from the main repo), it just should work without any extra steps. And docker caches image layers. If there is nothing to rebuild, this should be done in seconds.

Reply:
You can change "run-tests" as you like and also with build dependency.